### PR TITLE
Rspec tests

### PR DESCRIPTION
**Pull request by FelipeEnne**

### CODE REVIEW

**Review by Redvanisation — CHANGES_REQUESTED** [review]
# Changes required  ♻️ 

### Hello team  🌟

**Good work on this project, but we need to fix the issues below before moving on ⬇️** 

1. **All the methods** in the projects need to be tested.

2. All tests should cover both positive and negative scenarios, E.g: User enters valid data and users enters invalid data.

### [OPTIONAL]: 
- Separate the classes into different files for each one of them.

### _After making the required changes please submit a new code review request_

### Good luck! 🍀

File: spec/tictactoe_spec.rb [hunk 1]
```diff
@@ -0,0 +1,100 @@
+# frozen_string_literal: true
+
+# rubocop: disable Style/GlobalVars
+
+require_relative '../lib/main_test'
+
+$coordinates = ''
+a = TicTacToe.new
+name1 = 'ilhan'
+name2 = 'felipe'
```
Comment (Redvanisation) [inline spec/tictactoe_spec.rb]: Put each one of those inside a `let` variable and use it within the class test.

**Review by Redvanisation — CHANGES_REQUESTED** [review]
### Good job so far! 👍 

#### However, you still should implement the following:
1.  Use all the message methods in the tests to test if they are working, for example, expect that the winner message gets displayed when a player wins the game.

2. Check my comment below about using `let` variables.

### _After making the required changes please submit a new code review request_

### Good luck! 🍀

**Comment by Redvanisation** [comment]
### The previous changes are required before requesting a new code review.

**Review by uche-inyama — APPROVED** [review]
## Approved

Hi Felipe!

Everything is fine.

#### ``` Have a great day ``` :slightly_smiling_face: 

``` Uchechukwu Henry Inyama```

**Pull request merged**